sycl/test/basic_tests/vectors/vectors.cpp

```diff
@@ -10,21 +10,48 @@
 
 #define SYCL_SIMPLE_SWIZZLES
 #include <CL/sycl.hpp>
-using namespace cl::sycl;
 
-void check_vectors(int4 a, int4 b, int4 c, int4 gold) {
-  int4 result = a * (int4)b.y() + c;
+void check_vectors(cl::sycl::int4 a, cl::sycl::int4 b, cl::sycl::int4 c,
```

Comment:
I suggest dropping `cl::` prefix for all instances of `cl::sycl::` namespace qualifier.

E.g. `cl::sycl::int4` -> `sycl::int4`.